hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsBlobClient.java

```diff
@@ -889,7 +1144,7 @@ public AbfsRestOperation read(final String path,
     }
 
     final AbfsUriQueryBuilder abfsUriQueryBuilder = createDefaultUriQueryBuilder();
-    String sasTokenForReuse = appendSASTokenToQuery(path, SASTokenProvider.READ_OPERATION,
+    String sasTokenForReuse = appendSASTokenToQuery(path, SASTokenProvider.FIXED_SAS_STORE_OPERATION,
```

Comment:
Similar to the above LIST case, could you explain the reason for replacing everywhere "FIXED_SAS_STORE_OPERATION"

Reply:
"This is because the driver currently only supports fixed SAS for FNS Blob. This will be updated to the exact operation once user delegation SAS support is added."